hildon/hildon-color-chooser.c

```diff
@@ -109,19 +109,19 @@ init_borders                                    (GtkWidget *w,
                                                  GtkBorder *inner, 
                                                  GtkBorder *outer);
 
-inline void 
+static inline void 
```

Comment:
Please remove the trailing space in the lines you modified in this commit.
It's not your fault, but if you touch these lines, it's better to be fixed.

Reply:
I'll just sanitize all trailing whitespaces here, ok?

Reply:
If you do it in separate commit, it would be fine.

Reply:
Will do, once everything else is fine.

Reply:
Ok. Please let me know, when you are ready for a new review.